src/utils/bundleReport.js

```diff
@@ -0,0 +1,95 @@
+const path = require('path');
+const prettifyTime = require('./prettifyTime');
+const sizeTypes = ['B', 'KB', 'MB', 'GB'];
```

Comment:
What about using `KiB`, `MiB`, `GiB` or changing the factor to 1000 instead of 1024, for correctness?

Reply:
I'm not sure, this is the correct notation as used in almost every OS

Reply:
I know that Windows uses the wrong KB notation, and that most Linux distros use KiB notation. Don't know about Mac, Android, iOS and others.
(nice!)